packages/admin/cms-admin/src/pages/SaveConflictDialog.tsx

```diff
@@ -84,12 +149,21 @@ function SaveConflictDialog({ open, onClosePressed, onDiscardChangesPressed }: S
                 >
                     <FormattedMessage
                         id="comet.saveConflictDialog.actionButtons.openCurrentVersionInNewTab"
-                        defaultMessage="Open current version in new Tab"
+                        defaultMessage="View the other changes in a new Tab"
                     />
                 </Button>
             </DialogActions>
         </Dialog>
     );
 }
 
+const StyledList = styled(List)`
+    list-style-type: disc;
+    & .MuiListItem-root {
+        display: list-item;
+        padding-left: 0;
+    }
```

Comment:
Can you create a separate styled-component for this that extends from `ListItem`?

Reply:
Done